src/Routes.js

```diff
@@ -31,36 +30,34 @@ const AppRoutes = () => {
     'sed-frontend.activationKeysDetailsPage'
   );
   return (
-    <Authentication>
-      <Routes>
-        {keyDetailsIsEnabled && (
-          <Route
-            path={paths.activationKey}
-            element={
-              <SuspenseWrapped>
-                <ActivationKey />
-              </SuspenseWrapped>
-            }
-          />
-        )}
+    <Routes>
+      {keyDetailsIsEnabled && (
         <Route
-          path={paths.activationKeys}
+          path={paths.activationKey}
           element={
             <SuspenseWrapped>
-              <ActivationKeys />
+              <ActivationKey />
             </SuspenseWrapped>
           }
         />
-        <Route
-          path={paths.connector}
-          element={
-            <SuspenseWrapped>
-              <Dashboard />
-            </SuspenseWrapped>
-          }
-        />
-      </Routes>
-    </Authentication>
+      )}
+      <Route
+        path={paths.activationKeys}
+        element={
+          <SuspenseWrapped>
+            <ActivationKeys />
+          </SuspenseWrapped>
+        }
+      />
+      <Route
+        path={paths.connector}
```

Comment:
I am not sure how exactly this PR is causing it, but if I navigate by clicking on "View log" in the upper top corner, I get an error
```
(0 , react_router_dom__WEBPACK_IMPORTED_MODULE_4__.useHistory) is not a function
```

Reply:
it's not reproducible on stage c.r.c. (master version)